lib/rules/no-url-domains.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var helpers = require('../helpers'),
+    url = require('url');
+
+module.exports = {
+  'name': 'no-url-domains',
+  'defaults': {},
+  'detect': function (ast, parser) {
+    var result = [];
+
+    ast.traverseByType('uri', function (uri) {
+      uri.traverse(function (item) {
+        if (item.is('string')) {
+          var stripped = helpers.stripQuotes(item.content),
+              parsedUrl = url.parse(stripped, false, true);
+
+          if (parsedUrl.host && parsedUrl.protocol !== 'data:') {
```

Comment:
What about `//assets.example.com/some-font.ttf` ? It fails on this and I think it shouldn't.